File: Quizify-FrontEnd/src/app/quizify/tsclasses/question.ts
import { Category } from './category';
import { Admin } from './admin';
import { Topic } from './topic';


export class Question {

     id: string;
     category: Category;
     topic: Topic;
     level: string;
     type: string;
     statement: string;
     options: string[];
    answer: string;
     admin: Admin;

}

File: signle-player-engine/src/main/java/com/stackroute/quizify/signleplayerengine/domain/Question.java
package com.stackroute.quizify.signleplayerengine.domain;

import lombok.Data;
import org.springframework.data.annotation.Id;

import java.util.List;
@Data
public class Question {
@Id
    private String id;
    private Category category;
    private Topic topic;
    private String level;
    private String genre;
    private String type;
    private String statement;
    private List<String> options;
    private long timeStamp;
    private String answer;
    private String playeranswer;
    private Admin admin;
}
